src/pages/users/users-page/components/admin-tab/admin-tab.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import { push } from 'connected-react-router';
+import { useStyles } from './admin-tab.styles';
+import TableContainerGenerator from '../../../../../containers/table-container-generator';
+import TableContainerRow from '../../../../../containers/table-container-row';
+import { userRoleTranslations } from '../../../../../translations/user.translations';
+import { config } from '../../../../../configs';
+
+const { routes } = config.app;
+
+const pathToRegisterAdminPage = routes.pathToRegisterAdmin;
+
+const tableHeaders = config.tableHeadRowTitles.users.adminTab;
+const { CREATE_SPECIAL_USER } = config.buttonTitles;
+
+const AdminTab = (props) => {
+  const { list, onDelete } = props;
+  const styles = useStyles();
+  const dispatch = useDispatch();
+
+  const adminItems = list.map((userItem, index) => (
+    <TableContainerRow
+      key={index}
```

Comment:
Don't use index for key, use something stable like **id** or **name**

Reply:
fixed